README.md

```diff
@@ -201,18 +217,18 @@ There are two ways of running Manufacturer Component Sample:
 
 ```
 $ cd <fido-iot-src>/service/component-samples/manufacturer
-$ mvn -Dmanufacturer_di_port=<manufacturer-server-port> -Dmanufacturer_database_connection_url=<jdbc-url> -Dmanufacturer_database_username=<manufacturer-database-username> -Dmanufacturer_database_password=<manufacturer-database-password> -Dmanufacturer_database_port=<manufacturer-server-database-port> -Dcatalina_home=<path-to-catalina-home> exec:java
+$ mvn -Dmanufacturer_di_port=<manufacturer-server-port> -Dmanufacturer_database_connection_url=<jdbc-url> -Dmanufacturer_database_username=<manufacturer-database-username> -Dmanufacturer_database_password=<manufacturer-database-password> -Dmanufacturer_database_port=<manufacturer-server-database-port> -Dcatalina_home=<path-to-catalina-home> -Dmanufacturer_keystore_password=<manufacturer-keystore-password> -Dmanufacturer_api_user=<manufacturer-api-user> -Dmanufacturer_api_password=<manufacturer-api-password> exec:java
 ```
 
 ### FIDO IoT Manufacturer REST APIs
 
-1. POST /api/v1/assign/<guid>/<id>
+| Operation                      | Description                        | Path/Query Parameters    | Content Type   |Request Body  | Response Body |
+| ------------------------------:|:----------------------------------:|:------------------------:|:--------------:|-------------:|--------------:|
+| POST /api/v1/assign/?id=<customer_id>&guid=<guid> | Assigns customer ID to voucher having the input GUID. | Query - id: Customer ID, guid = Device GUID | | | |
```

Comment:
&guid=<guid>; "\<guid\>" is not rendering in UI. Try adding "\\" before "<" & ">".